corehq/motech/repeaters/management/commands/create_missing_repeat_records.py

```diff
@@ -0,0 +1,81 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.es.forms import FormES
+from corehq.motech.repeaters.dbaccessors import (
+    get_domains_that_have_repeat_records,
+    get_repeat_records_by_payload_id,
+    get_repeaters_by_domain,
+)
+from corehq.motech.repeaters.models import FormRepeater, ShortFormRepeater
+from corehq.util.argparse_types import date_type
+
+
+def create_missing_repeat_records(startdate, enddate, domain, detailed_count=False, should_create=False):
+    if domain:
+        domains_with_repeaters = [domain]
+    else:
+        domains_with_repeaters = get_domains_that_have_repeat_records()
+    missing_records_per_domain = {}
+    for domain in domains_with_repeaters:
+        forms = get_forms_in_domain_between_dates(domain, startdate, enddate)
+        form_repeaters = [repeater for repeater in get_repeaters_by_domain(domain)
+                          if isinstance(repeater, FormRepeater) or isinstance(repeater, ShortFormRepeater)]
+        count_missing = 0
+        for form in forms:
+            if detailed_count:
+                count_missing += create_missing_repeat_records_for_form(domain, form_repeaters, form,
+                                                                        should_create)
+            else:
+                count_missing += count_missing_repeat_records_for_form(domain, form_repeaters, form)
+        if count_missing > 0:
+            missing_records_per_domain[domain] = count_missing
+
+    print(f"Missing records per domain:\n {missing_records_per_domain}")
+
+
+def count_missing_repeat_records_for_form(domain, repeaters, form):
+    count_missing = 0
+    repeat_records = get_repeat_records_by_payload_id(domain, form['_id'])
+    if len(repeat_records) != len(repeaters):
+        count_missing += len(repeaters) - len(repeat_records)
+    return count_missing
+
+
+def create_missing_repeat_records_for_form(domain, repeaters, form, should_create):
+    count_missing = 0
+    repeat_records = get_repeat_records_by_payload_id(domain, form['_id'])
+    for repeater in repeaters:
+        found = False
+        for repeat_record in repeat_records:
+            if repeat_record.repeater_id == repeater.get_id:
+                found = True
+                break
+
+        if not found:
```

Comment:
omg how often do you find a legitimate use for [Python's `for ... else`](https://book.pythontips.com/en/latest/for_-_else.html#else-clause)?! :grinning: